pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -2481,6 +2481,56 @@ status:
 	}
 }
 
+func TestReconcileWithNewTimeouts(t *testing.T) {
```

Comment:
nit: "new" becomes stale pretty quickly :) I'd just leave it as "testReconcileWithTimeouts"

Reply:
yeah, actually that time my VSCode gave me some error with the suggested name but now it seems to work fine. Will update it.